INSTALL.md

```diff
@@ -1,23 +1,112 @@
+
+- #### Linux installation
+
+    The command below will install all the packages needed.
+
+    ```Shell
+    sudo apt-get install python3-dev gcc g++ cmake libboost-all-dev
+    ```
+    Newer versions of boost can be downloaded in:
+
+    http://www.boost.org/users/download/.
+
+- #### Windows installation
+
+    - Visual Studio
+
+        *Visual Studio* is the only compiler officially supported to build *Kratos* under *Windows*. The minimium required version is Visual Studio 2017, but we recommend to use Visual Studio 2019 or higher. 
+
+        * [Download Visual Studio](https://visualstudio.microsoft.com/en/thank-you-downloading-visual-studio/?sku=Community&rel=15)
+
+        Since *Visual Studio* is a multi-language IDE, some distributions come without C++ compiler. Please, make sure that you can create a C++ project before continuing, in case C++ packages were missing you will be prompt to download them.
+
+    - CMake
+        * [Download CMake](http://cmake.org/download/)
+
+        Once installing, please <span style="color:red"> do not forget to mark the option: '''"Add CMake to the system PATH for all users"'''</span>
+
+        Minimum required version: CMake 3.14
+        
+    - Python
+
+        You will need at least *Python* 3.5 (recommended 3.7/3.8) in your computer in order to compile *Kratos*. You can download python from its official webpage:
+
+        * [Download Python](http://www.python.org/downloads/)
 
+        Please, take special care to download a installer that suits your desired architecture <span style="color:red">x86 for 32 bits</span>  compilations and <span style="color:red">x86_64 for 64 bits</span>  compilations. Otherwise it won't work.
+
+    - Boost
+
+        The next step will consist in obtain Boost. *Kratos Multiphysics* needs *Boost* libraries to support some of its functions. You can use any version from `version 1.67` onward.
+
+        * [Download Boost](http://www.boost.org/users/download/)
+
+        Extract boost, and note the path as it will be needed in the configure stage to set the environmental variable `BOOST_ROOT`.
+
+
+### Specific Application Dependencies
+
+Specific compilation information about applications can be found in their own directories:
+- [EigenSolvers Application (only required if MKL is used)](applications/EigenSolversApplication/README.md#build-instructions)
+- [ExternalSolvers Application](applications/ExternalSolversApplication/README.md#installing-blas-and-lapack)
+- [HDF5 Application](applications/HDF5Application/README.md#build-instructions)
+- [Metis Application](applications/MetisApplication/README.md)
+- [MultilevelMontecarlo Application](applications/MultilevelMonteCarloApplication/README.md#external-libraries)
+- [Poromechanics Application](applications/PoromechanicsApplication/README.md#how-to-use-mpi-in-poromechanics-application)
+- [Trilinos Application (Aditional notes)](applications/TrilinosApplication/README.md#notes-for-compilation)
```

Comment:
I would remove this entirely with sth like this:
```suggestion
Some applications have additional dependencies. Please check the `README` files of the applications that are compiled
```

Reply:
This is far easier to maintain, imagine we add another app that has a dependency, then we would need to add it here
same with optional dependencies

=> Better have it in the apps themselves